src/status.rs

```diff
@@ -453,14 +455,16 @@ impl StatusBuilder {
     }
 
     pub fn build(self) -> Result<Status, String> {
-        Ok(Status::new(
-            self.space,
-            self.logo.ok_or("logo missing")?,
-            self.url.ok_or("url missing")?,
-            self.location.ok_or("location missing")?,
-            self.contact.ok_or("contact missing")?,
-            self.issue_report_channels,
-            ))
+        Ok(Status {
+            api: "0.13".into(), // TODO: Deduplicate
```

Comment:
That's true, now that we have custom serialization anyways.

On the other hand, it's part of the response. If for example the validator deserializes a response, it wants to be able to access that field, right?